### ISSUE TITLE
Did you try to fine-tune transformers LM with Ranger?

### ISSUE THREAD

**Issue by avostryakov** [lessw2020/Ranger-Deep-Learning-Optimizer#13]
Recent transformers architectures are very famous in NLP: BERT, GPT-2, RoBERTa, XLNET. Did you try to fine-tune them on some NLP task? If so, what was the best Ranger hyper-parameters and learning rate scheduler?

**Comment by LifeIsStrange** [comment]
Testing for XLnet should be prioritarised as it is the current best state of the art.
ERNIE 2.0 would be interesting too.

**Comment by JohnGiorgi** [comment]
@avostryakov I tried fine-tuning a BERT based model for joint NER and relation classification. It performs about ~1.5% worse for my tasks than the AdamW implementation in [Transformers](https://huggingface.co/transformers/):

__AdamW__

- lr: 3e-5
- betas: (0.9, 0.999)
- eps: 1e-6
- weight_decay: 0.1
- correct_bias: True

__Ranger__

- lr: 3e-4
- betas: (0.95, 0.999)
- eps: 1e-6
- weight_decay: 0.1

It is possible that with more tunining I might be able to close the gap.

**Comment by lessw2020** [comment]
I'm working with DETR which is object detection with transformer internally and will test it out there soon.  
Note that Ranger now has GC (gradient centralization) and will be interesting to see if that helps for transformers.

**Comment by hiyyg** [comment]
How does ranger perform for Detr?
